src/LanguageServer/Protocol/Handler/RelatedDocuments/RelatedDocumentsHandler.cs

```diff
@@ -41,19 +41,19 @@ public TextDocumentIdentifier GetTextDocumentIdentifier(VSInternalRelatedDocumen
     public async Task<VSInternalRelatedDocumentReport[]?> HandleRequestAsync(
         VSInternalRelatedDocumentParams requestParams, RequestContext context, CancellationToken cancellationToken)
     {
-        context.TraceInformation($"{this.GetType()} started getting related documents");
+        context.TraceDebug($"{this.GetType()} started getting related documents");
```

Comment:
Not sure why this is including the class name if it's sealed....does the method name now make it sufficiently clear since we log that too?

Reply:
yup.  actually I think I'm going to remove this entirely, it isn't particularly useful to log starting a request since the queue already does